### PR TITLE
add sendHttpRequest to work with http

### PR DESCRIPTION
**Pull request by nomid0612**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@qavajs/api-service",
+  "version": "0.0.4",
```
Comment (AlexGalichenko) [inline package.json]: 0.0.1

File: index.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
+/* eslint-disable import/no-unresolved */
+require('./lib/httpRequest.js');
```
Comment (AlexGalichenko) [inline index.js]: need to add re-export to allow import

```
const httpRequest = require('./lib/httpRequest.js');
module.exports = { httpRequest }
```

**Review by AlexGalichenko — CHANGES_REQUESTED** [review]

File: index.js (hunk 2 above)
Comment (nomid0612) [reply index.js]: fixed

File: package.json (hunk 1 above)
Comment (nomid0612) [reply package.json]: fixed

**Review by KirillBogdanets — APPROVED** [review]

**Review by AlexGalichenko — APPROVED** [review]

**Pull request merged**